modules/apps/notification/notification-rest-impl/rest-openapi.yaml

```diff
@@ -63,6 +97,105 @@ info:
+                - in: query
+                  items:
+                      type: string
+                  name: aggregationTerms
+                  schema:
+                  type: array
+                - in: query
+                  name: filter
+                  schema:
+                      type: string
+                - in: query
+                  name: page
+                  schema:
+                      type: integer
+                - in: query
+                  name: pageSize
+                  schema:
+                      type: integer
+                - in: query
+                  name: search
+                  schema:
+                      type: string
+                - in: query
+                  name: sort
+                  schema:
+                      type: string
+                - in: header
+                  name: Accept-Language
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/NotificationQueueEntry"
+                                type: array
+                        application/xml:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/NotificationQueueEntry"
+                                type: array
+            tags: ["NotificationQueueEntry"]
+    "/notification-queue-entry/{notificationQueueEntryId}":
+        delete:
+            operationId: deleteNotificationQueueEntry
+            parameters:
+                - in: path
+                  name: notificationQueueEntryId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+            responses:
+                204:
+                    content:
+                        application/json: {}
+                        application/xml: {}
+            tags: ["NotificationQueueEntry"]
+        get:
+            operationId: getNotificationQueueEntry
+            parameters:
+                - in: path
+                  name: notificationQueueEntryId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/NotificationQueueEntry"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/NotificationQueueEntry"
+            tags: ["NotificationQueueEntry"]
+    "/notification-queue-entry/{notificationQueueEntryId}/resend":
```

Comment:
@gustavolimav can you remove this one too?

Reply:
Ok